gordo/workflow/config_elements/normalized_config.py

```diff
@@ -25,6 +25,7 @@ class NormalizedConfig:
 
     DEFAULT_CONFIG_GLOBALS = {
         "runtime": {
+            "reporters": [],
```

Comment:
Ok, so it gets patched from here, so the validation insertion is just another check? It does seem a data class would be pretty nice, as it would address the patching stuff too.